OpenAI_Tests/ChatEndpointTests.cs

```diff
@@ -149,6 +153,54 @@ public void ChatBackAndForth(string model)
 			Assert.IsNotEmpty(res);
 			Assert.AreEqual("No", res.Trim());
 		}
+		[Test]
+		public async Task SummarizeFunctionResult()
+		{
+			try
+			{
+				var api = new OpenAI_API.OpenAIAPI();
+
+				var conversation = api.Chat.CreateConversation(new ChatRequest { Model = Model.ChatGPTTurbo0613 });
+				conversation.AppendUserInput("What is the weather like in Boston?");
+
+                var initialAssistantMessage = new ChatMessage
+                {
+                    Role = ChatMessageRole.Assistant,
+                    Content = "(null)",
+                    Function_Call = new Function_Call()
+                    {
+                        Name = "get_current_weather",
+                        Arguments = "{ \"location\": \"Boston, MA\"}"
+                    }
+                };
+                conversation.AppendMessage(initialAssistantMessage);
+                var functionMessage = new ChatMessage
+                {
+                    Role = ChatMessageRole.Function,
+                    Name = "get_current_weather",
+                    Content = "{\"temperature\": \"22\", \"unit\": \"celsius\", \"description\": \"Sunny\"}"
+                };
+                var functionList = new List<Function>
+                {
+                    BuildFunctionForTest()
+                };
+				conversation.AppendMessage(functionMessage);
+                conversation.RequestParameters.Functions = functionList;
+				conversation.RequestParameters.Temperature = 0;
+
+
+				var response = await conversation.GetResponseFromChatbotAsync();
+				
+
+				Assert.AreEqual(response, "The current weather in Boston is sunny with a temperature of 22 degrees Celsius.");
```

Comment:
These parameters are reversed. The expected response should come first.
```suggestion
				Assert.AreEqual("The current weather in Boston is sunny with a temperature of 22 degrees Celsius.", response);
```